.circleci/config.yml

```diff
@@ -610,14 +610,14 @@ jobs:
             END_TIME=$(date +%s)
 
             if << pipeline.parameters.randomize-aws-credentials >> ; then
-              echo "export TINYBIRD_WORKFLOW=tests_circleci_ma_mr" >> $BASH_ENV
+              export TINYBIRD_WORKFLOW=tests_circleci_ma_mr
             else
-              echo "export TINYBIRD_WORKFLOW=tests_circleci" >> $BASH_ENV
+              export TINYBIRD_WORKFLOW=tests_circleci
             fi
 
             # Build the payload
             echo '{"workflow": "'$TINYBIRD_WORKFLOW'", "attempt": 1, "run_id": "'$CIRCLE_WORKFLOW_ID'", "start": '$START_TIME', "end": '$END_TIME', "commit": "'$CIRCLE_SHA1'", "branch": "'$CIRCLE_BRANCH'", "repository": "'$CIRCLE_PROJECT_USERNAME'/'$CIRCLE_PROJECT_REPONAME'", "outcome": "'$OUTCOME'", "workflow_url": "'$CIRCLE_BUILD_URL'"}' > stats.json
-            echo 'Sending: '$(cat stats/stats.json)
+            echo 'Sending: '$(cat stats.json)
```

Comment:
Since the runner has `jq` installed, we could use it here as well to get an easier to read output

Reply:
Honestly, I  would like to avoid any overhead / additional tooling here. For example, I don't want this to fail because there is an issue with the JSON being sent. It really should just print the file which is about to be sent.